### PR TITLE
[ASAbsoluteLayoutSpec] Add ASAbsoluteLayoutSpecSizing

### PR DESCRIPTION
**Pull request by maicki**
Add ability for ASAbsoluteLayoutSpec to grow as far as possible or shrink to fit it's children

### CODE REVIEW

**Comment by maicki** [comment]
It does not change the behavior of `ASAbsoluteLayoutSpec` if you don't see it as a find a replace for `ASStaticLayoutSpec`. We need to add that to the conversion guide to set the flag if you want the behavior from `ASStaticLayoutSpec`.

`ASAbsoluteLayoutSpec` actually works now the same way in terms of sizing like `ASCenterLayoutSpec` and other layout specs. It just adds the ability to size itself to fit it's content.

File: AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h [hunk 1]
```diff
@@ -10,16 +10,33 @@
...
 /**
+ How much space will the spec taken up
+ */
+@property (nonatomic, assign) ASAbsoluteLayoutSpecSizing sizing;
+
+/**
+ @param sizing How much space the spec will take up
+ @param children Children to be positioned at fixed positions
+ */
++ (instancetype)absoluteLayoutSpecWithSizing:(ASAbsoluteLayoutSpecSizing)sizing children:(NSArray<id<ASLayoutElement>> *)children AS_WARN_UNUSED_RESULT;
```
Comment (garrettmoon) [inline AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h]: should this be `absoluteLayoutSpecWithChildren:sizing:`? To match the next one?

Comment (maicki) [inline AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h]: All the other layout spec have the children at the end and I think it's the right approach also here.

Comment (garrettmoon) [inline AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.h]: That's sounds right to me.

**Pull request merged**